### ISSUE TITLE
Choropleth plot doesn't appear to have geojson prop

### ISSUE THREAD

**Issue by geoguide** [plotly/react-plotly.js#304]
When I'm looking at the documentation it looks like I can set the `locationmode: geojson-id` and for the `type: choropleth` and should then be able to set the `geojson` prop to a geojson object, but that property does not appear to exist on the component, it certainly does not on the typescript. Any advice?

**Comment by alexcjohnson** [comment]
Yes, that should work. What plotly.js version are you using this with? If it's a recent one you can see it when you hover on the logo in the toolbar:
<img width="208" alt="Screenshot 2022-12-01 at 18 29 02" src="https://user-images.githubusercontent.com/2678795/205180846-f4a073fd-ea77-4b33-b71c-d3921a85301e.png">

I don't know much about the typescript definitions and how up-to-date they are, those are maintained by a third party, not Plotly.

**Comment by geoguide** [comment]
Thanks

I'm using 2.16.3

I can try to bypass the typescript to get it to work, it's hard when it's not behaving as expected and not throwing errors.
